packages/test-utils/src/wrapper.js

```diff
@@ -569,10 +569,24 @@ export default class Wrapper implements BaseWrapper {
       tagName === 'TEXTAREA' ||
       tagName === 'SELECT'
     ) {
-      const event = tagName === 'SELECT' ? 'change' : 'input'
       // $FlowIgnore
       this.element.value = value
-      this.trigger(event)
+
+      if (tagName === 'SELECT') {
+        this.trigger('change')
+      } else {
+        this.trigger('input')
+      }
+
+      // for v-model.lazy, we need to trigger a change event, too.
+      // $FlowIgnore
+      if (
+        tagName === 'INPUT' &&
```

Comment:
`.lazy` can also be used on `TEXTAREA`.

Reply:
oops, updated! all green locally. CI failing though 🤔